synapse/swarm/opers/common.py

```diff
@@ -23,6 +27,34 @@ def runSyntaxCheck(self):
         pass
 
     def run(self):
+        debug_count = self.query.opt('debug:count')
```

Comment:
since the query is the one with all the opts/state/recording ( and an Oper really shouldn't need to be aware of this ) i think we should move the stats gathering code to the execute() method within the query.  Then it would also only need to pull those opts once etc as well.